### PR TITLE
fix(GiniCaptureSDK): Fix some spelling mistakes in the new UI/UX SDK iOS

### PR DESCRIPTION
**Pull request by ValentinaIancu-Gini**
Fix spelling mistakes:
- QR code/ PDF Document - German/English version
- correct usage of the formal form in Deutsch
- Photo/Foto - English/German version
- correct some messages to be more grammatically correct

PIA-4387

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -47,12 +47,12 @@
 "ginicapture.camera.flashButtonLabel.Off" = "Off";
 "ginicapture.camera.thumbnail.Voice.Over" = "Photo review";
 "ginicapture.camera.capturebutton" = "Take picture";
-"ginicapture.camera.infoLabel.invoice.and.qr" = "Scan an invoice or a QR code";
+"ginicapture.camera.infoLabel.invoice.and.qr" = "Scan invoice or QR code";
 "ginicapture.camera.infoLabel.only.invoice" = "Scan an invoice";
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/en.lproj/Localizable.strings:51]: Shall we remove the article here?
```suggestion
"ginicapture.camera.infoLabel.only.invoice" = "Scan invoice";
```

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -47,12 +47,12 @@
...
 "ginicapture.camera.infoLabel.only.qr" = "Scan a QR code";
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/en.lproj/Localizable.strings:52]:
```suggestion
"ginicapture.camera.infoLabel.only.qr" = "Scan QR code";
```
@ValentinaIancu-Gini  shall we remove the article here?

**Review by zladzeyka — COMMENTED** [review]
@ValentinaIancu-Gini Looks great 🎉
I left a few comments and added the question to the chat 😊

**Comment by ValentinaIancu-Gini** [comment]
@zladzeyka all the comments are fixed! Thanks.

**Review by zladzeyka — APPROVED** [review]
👍🏻

**Pull request merged**